mkdocs.yml

```
@@ -654,6 +659,8 @@ nav:
                 - URL Sort Clauses: search/url_search_reference/url_search_sort_clauses.md
                 - Id Sort Clause: search/url_search_reference/id_url_sort_clause.md
                 - Url Sort Clause: search/url_search_reference/url_url_sort_clause.md
+            - Activity Log Sort Clauses:
+                - Activity Log Sort Clauses: search/activity_log_search_reference/activity_log_sort_clauses.md
```

Too short to be a rubric
```suggestion
            - Activity Log Sort Clauses: search/activity_log_search_reference/activity_log_sort_clauses.md
```

---

Done in 30606858c3499a0fcdaaa43682d243cf2c96052c